python/pfs/drp/stella/cosmicray.py

```diff
@@ -331,8 +341,9 @@ def run(self, exposures: List["Exposure"]) -> Struct:
         clean = self.calculateCleanImage([exposure.getMaskedImage() for exposure in exposures])
         result.mergeItems(clean, "image", "scales")
         psf = self.config.modelPsf.apply()
+        rms = self.calculateChiRms(exposures, clean)
```

Comment:
This feels like a workaround for bad noise models. Why are the noise models bad?